### ISSUE TITLE
The server that the API documentation is on seems to be down

### ISSUE THREAD

**Issue by markhorgan** [RestKit/RestKit#2224]
restkit.org where the API documentation is held seems to be down for a while.

Just want to say that this is a great project, I rely on it a lot in my projects.

**Comment by markhorgan** [comment]
Any reason why this issue was closed without a comment? Are you able to access the API documentation or is this not the place to raise issues on this?

**Comment by segiddins** [comment]
It's something we know is down, we're working to get it back up. I try and keep open issues for actual issues with RestKit itself.

**Comment by blakewatters** [comment]
The company that had been providing the hosting of the site was acquired and the hosting was shut down. I understood the change to being DNS level only but it seems the instance went away. I’m trying to find a backup that I captured awhile ago to restore the site on one of my own instances, but thus far I haven’t been able to figure out where I archived the backup (I know its around here somewhere).

In the meantime, I’ve done a temporary redirect of restkit.org back to the Github site and I’m updating the API docs to point to CocoaDocs instead.
